src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -147,6 +148,7 @@ public function editAction(Post $post, Request $request)
         if ($form->isSubmitted() && $form->isValid()) {
             $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
             $entityManager->flush();
+            $this->getDoctrine()->getRepository(Tag::class)->cleanUnusedTags();
```

Comment:
Personal choice mainly (I understand if you close the issue).

Having tags attached to nothing could create some problem if you choose to create a tagcloud later for instance.